lib/signer.js

```diff
@@ -41,9 +41,13 @@ function FormatAuthz(prefix, params) {
     var value = params[param];
     if (value === undefined)
       continue;
-    assert.string(value, 'params.' + param);
+    if (typeof value === 'number') {
+      authz += prefix + sprintf('%s=%s', param, value);
```

Comment:
Shouldn't this be `%s=%d` or `%s=%u`?

Reply:
It works both (`%s` and `%d`), since `%s` automatically converts the number to a string (see https://nodejs.org/api/util.html#util_util_format_format_args). `%u` is not documented, I will update this to `%s` to better signal that value is a number.